reports/GoodsAndServiceTax/GSTR1.js

```diff
@@ -14,10 +14,10 @@ class GSTR1 extends BaseGSTR {
     const data = await this.getCompleteReport('GSTR-1', filters);
 
     const conditions = {
-      B2B: row => row.gstin,
+      'B2B': row => row.gstin,
       'B2C-Large': row => !row.gstin && !row.inState && row.invAmt >= 250000,
-      'B2C-Small': row =>
-        !row.gstin && (row.inState || (row.inState && row.invAmt < 250000))
+      'B2C-Small': row => !row.gstin && (row.inState || (row.inState && row.invAmt < 250000)),
+      'Nil Rated, Exempted and Non GST supplies': row => (row.rate === 0), // this takes care of both nil rated, exempted goods
```

Comment:
Can this have a shorter name that doesn't break the UI? 😅

Reply:
I think we'll have to keep this and update the UI to accommodate this :sweat_smile.

P.S. We are not exporting this for reports but this is a useful filter